### DISCUSSION TITLE
github pages

### DISCUSSION THREAD

**Discussion by blake41** [mmistakes/minimal-mistakes#2909] [General]
I added my own main.scss file to make some small edits to the css.  This works no problem locally but is not getting pulled in on github pages.  Any ideas what I might be missing?

**Comment by iBug** [comment]
Would you please provide some context? It's hard to diagnose anything without actual code.

**Reply by blake41** [reply]
sure, this is my code https://github.com/blake41/cryptonyc
locally my css edits are working, deployed at https://blake41.github.io/cryptonyc I don't see the css working

**Reply by iBug** [reply]
Your code looks OK. Maybe try clearing browser cache?

**Reply by blake41** [reply]
ahh, maybe that was it, looks fine now.  i'm an idiot, thanks so much for your help
